corehq/util/dates.py

```diff
@@ -18,3 +18,28 @@ def get_timestamp(date):
 
 def get_timestamp_millis(date):
     return 1000 * get_timestamp(date)
+
+
+def safe_strftime(val, fmt):
```

Comment:
That should really be part of the comment, I'll add it, and remove the original function.